ruby-challenges.rb

```diff
@@ -32,3 +61,17 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+
```

Comment:
ensure you are looking at the output. You may have to refactor on what is being called a palindrome.